autodock/__init__.py

```diff
@@ -129,16 +129,18 @@ def addAutoDockGPUPackage(cls, env, default=True):
 		# Instantiating the install helper
 		installer = InstallHelper(ADGPU_DIC['name'], packageHome=cls.getVar(ADGPU_DIC['home']), packageVersion=ADGPU_DIC['version'])
 
-		# Getting Nvidia card data
-		compCapDic = cls.getNVIDIACompCapDic()
-		nvidiaName = cls.getNVIDIAName()
-		compCap = compCapDic[nvidiaName] if nvidiaName in compCapDic else None
-		targetsFlag = f' TARGETS={compCap}' if compCap else ''
-
-		# Installing package
-		installer.getCloneCommand(cls.getAutoDockGPUGithub(), binaryFolderName=cls._atdgpuBinary, targeName='ATDGPU_CLONED')\
-			.addCommand(f'cd {cls._atdgpuBinary} && make DEVICE=GPU OVERLAP=ON{targetsFlag}', 'ATDGPU_COMPILED')\
-			.addPackage(env, dependencies=['git', 'make'], default=default, vars=enVars, updateCuda=True)
+		# Only install AutoDockGPU on systems with Nvidia GPUs
+		if cls.getGPUPlatform() == 'nvidia':
```

Comment:
I understand that here we could configure it to work with non-nvidia GPUs, but would AutoDockGPU even allow it? If so, we should add the elifs at some point

Reply:
Yes, how to do so is specified [here](https://github.com/ccsb-scripps/AutoDock-GPU/wiki/Guideline-for-users#compilation), but we would need an AMD GPU or a CPU accelerator card to test the installation, so we know it also works as expected, or if we need also to make some adjustments when running it.